src/rgw/rgw_cr_rados.h

```diff
@@ -248,7 +239,7 @@ int RGWSimpleRadosReadCR<T>::request_complete()
       return ret;
     }
     try {
-      bufferlist::iterator iter = bl.begin();
+      bufferlist::iterator iter = req->bl.begin();
```

Comment:
@cbodley should it be "auto iter" here ? also const iterator is not available for bufferlist here.

Reply:
good catch. might as well leave it as `bufferlist::iterator`